src/prx/user.py

```diff
@@ -106,3 +108,135 @@ def spp_pt_lsq(df, dx_convergence_l2=1e-6, max_iterations=10):
             n_iterations <= max_iterations
         ), "LSQ did not converge in allowed number of iterations"
     return x_linearization
+
+
+def first_position_from_georinex(filepath_obs, filepath_nav):
```

Comment:
Suggest something like `bootstrap_coarse_receiver_position`, that way the function name is independent of the RINEX parser we use.